CHANGELOG.md

```diff
@@ -38,6 +38,7 @@
 - \#2845 - Misleading error feedback in app creation modal on port conflict
 - \#2812 - Cannot change configuration of Marathon app after deployment
 - \#2821 - Tooltip content update issue
+- \#2972 - Add more inline task information
```

Comment:
I temporarily added this to 0.14.1 – i guess we'll have to create a dedicated section if we tag an 0.14.2